modules/artifact/src/test/java/org/mule/runtime/module/artifact/internal/descriptor/ArtifactTestingFeatures.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.internal.descriptor;
+
+import static java.util.Optional.ofNullable;
+
+import org.mule.runtime.api.config.Feature;
+import org.mule.runtime.core.api.config.FeatureFlaggingRegistry;
+
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+
+/**
+ * Features meant to be used in {@link Feature} flag related tests.
+ * 
+ * @see FeatureFlaggingUtils
+ * @see org.mule.runtime.api.config.FeatureFlaggingService
+ * @since 4.5.0
+ */
+public enum ArtifactTestingFeatures implements Feature {
```

Comment:
This class has been copied from [DeploymentTestingFeature](https://github.com/mulesoft/mule/pull/11686/files#diff-79c4c2e18c754d872d9a9f12dc8cfce5eb8db436e7e7f198678f2944626ee990).
This is because when we moved the `FeatureFlaggingUtilsTestCase` to this package we also needed to bring these features. But some of them (not all) were still needed for the tests in `deployment-impl`, this is why it wasn't just a move.